### PR TITLE
chore: update release procedure

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

+ Adds information about release model
+ Updates contributing instructions.

closes: #879

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added appropriate labels to the PR
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: CONTRIBUTING.md
Comment (adamewozniak) [inline CONTRIBUTING.md:95]:
```suggestion
All major changes related to major version update are first released for testnet. We use `-betaX` (eg `2.0.0-beta1`, `2.0.0-beta2` ...) releases for testnet. Once the code is stabilized we create a release candidate (eg `2.0.0-rc1`). If no issues are found the latest release candidate become the major release.
```

File: CONTRIBUTING.md
Comment (adamewozniak) [inline CONTRIBUTING.md:87]: Would be wise for us to state that we have not followed this guideline in the past, and at what point we are committing to this (e.g., once v3.0.0 is released)

**Review by adamewozniak — APPROVED** [review]
This is awesome 🎊 I recommend getting a couple more approvals before merging

File: CONTRIBUTING.md
Comment (robert-zaremba) [reply CONTRIBUTING.md:87]: You mean the `2.0` release? nah... it's still a full release for umeemania, and v3.0 contains new features and breaking changes.

Comment (adamewozniak) [reply CONTRIBUTING.md:87]: But the patch version update 2.0.1 does have new features, not just containing hot fixes, and we've not used the "beta1/rc" syntax at all. It's alright as long as the team is aligned 👍

**Review by toteki — APPROVED** [review]
+1 approval

There's also a linter issue, so this will be good to merge once that's passing

File: CONTRIBUTING.md
Comment (robert-zaremba) [reply CONTRIBUTING.md:87]: If you prefer, I can rephrase to:
 
> We follow semver (from v3.0.0):

Comment (adamewozniak) [reply CONTRIBUTING.md:87]: Just openede https://github.com/umee-network/umee/pull/917 to address 👍

**Pull request merged**
